tests/tests_spec.rb

```diff
@@ -3,29 +3,29 @@
 require 'rspec'
 require 'webmock/rspec'
 require 'json'
-WebMock.disable_net_connect!(:allow_localhost => true)
+WebMock.disable_net_connect!(allow_localhost: true)
 
 describe RosetteAPI do
 
   describe '.get_language' do
-    request_file = File.open('../mock-data/request/language.json')
+    request_file = File.read File.expand_path(File.join(File.dirname(__FILE__), '../mock-data/request/language.json'))
     before do
       stub_request(:post, 'https://api.rosette.com/rest/v1/info?clientVersion=1.0.2').
-          with(:body => "\"{\\\"body\\\": \\\"version check\\\"}\"",
-               :headers => {'Accept' => 'application/json',
+          with(body: "\"{\\\"body\\\": \\\"version check\\\"}\"",
```

Comment:
Prefer the `symbol: value` style for hashes than the older `:symbol => value` style.